doc/devel/coding_style_guideline.rst

```diff
@@ -35,6 +35,25 @@ compliance of your code with PEP8. Beyond the aspects checked, as a
 contributor to DIPY, you should try to ensure that your code, including
 comments, conform to the above principles.
 
+-------------------
+Cython coding style
+-------------------
+DIPY recommends the use of the standard Python
+`PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ style when writing
+`Cython <https://cython.org/>` code. The sole exception to these guidelines is
+the line length limitation recommendation: a line length of 120 characters is
+accepted for DIPY Cython code.
```

Comment:
Really?  Why?

Reply:
That was mentioned in #583 by @Garyfallidis . See [here](https://github.com/nipy/dipy/issues/583#issuecomment-76641756).

Reply:
+1 for keeping the pep8 recommendations:   line length limit = 79

Reply:
Removed the exception in commit 4f6e11c.